packages/mgt-components/src/components/mgt-person/strings.ts

```diff
@@ -5,7 +5,18 @@
  * -------------------------------------------------------------------------------------------
  */
 
+const availabilityMap = {
+  Available: 'Available',
+  Away: 'Away',
+  Busy: 'Busy',
+  DoNotDisturb: 'Do Not Disturb',
+  Offline: 'Offline',
+  BeRightBack: 'Be Right Back',
+  PresenceUnknown: 'Presence Unknown'
```

Comment:
Just a nit, I remember having to update to sentence casing a while back when working on documentation as it is the standard. Have a look at this https://learn.microsoft.com/en-us/style-guide/capitalization. Cc @sebastienlevert